src/components/Link/Link.stories.tsx

```diff
@@ -1,6 +1,12 @@
 import type { Meta, StoryObj } from '@storybook/react';
 
 import Link from './Link';
+import { DOWNLOAD_LINK_VARIANTS, LINK_VARIANTS } from './Link.types';
+
+/** Exclude download link variant options */
+const variantOptions = LINK_VARIANTS.filter(
+  (opt) => !DOWNLOAD_LINK_VARIANTS.includes(opt as (typeof DOWNLOAD_LINK_VARIANTS)[number])
+);
```

Comment:
These keeps "cta-download" out of the radio options for the standard Link component. The DownloadLink component requires an `href` and Link does not, so this will hopefully make it harder to misuse.